boards/m5stack/m5stack_cores3/Kconfig.defconfig

```diff
@@ -39,4 +39,11 @@ config DISPLAY_INIT_PRIORITY
 	default 83
 endif # DISPLAY
 
+if AUDIO_CODEC
+
+config AUDIO_CODEC_INIT_PRIORITY
+	default 80
```

Comment:
Would be nice to add a justification ("audio codec needs to be initialized before/after xxx") this way it is easier to maintain things later if more things get added